packages/b/boost/xmake.lua

```diff
@@ -78,6 +78,8 @@ package("boost")
     for _, libname in ipairs(libnames) do
         add_configs(libname,    { description = "Enable " .. libname .. " library.", default = (libname == "filesystem"), type = "boolean"})
     end
+    add_configs("nozstd", {description = "disable zstd for iostreams", default = true, type = "boolean"})
+    add_configs("nolzma", {description = "disable lzma for iostreams", default = true, type = "boolean"})
```

Comment:
and here

Reply:
ok. all neme changed. also  update all the if statement